README.md

```diff
@@ -1,120 +1,67 @@
-
-#### Using kubectl proxy and the proxy verb
-
-To access cluster-insight using kubectl proxy and the proxy verb:
-
-* Run kubectl proxy to expose the cluster on a local port, as described 
-[here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl_proxy.md). 
-* Assuming that you exposed the cluster on local port 8001 and that you're using API version v1, you should be able to find the cluster-insight service at `http://localhost:8001/api/v1/namespaces/default/services/cluster-insight`.
-* With the same assumptions, you should be able to access the service through the api-server using the proxy verb, at `http://localhost:8001/api/v1/proxy/namespaces/default/services/cluster-insight:cluster-insight/`.
-
-#### Using a load balancer
+```
+git clone https://github.com/google/cluster-insight --branch=v2
+cd cluster-insight/collector
 
-To access cluster-insight using a load balancer, set the service type to LoadBalancer in `./cluster-insight/install/cluster-insight-service.yaml`, as described [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#type--loadbalancer), and run the installation script. Depending on the platform, you may also have to create a firewall rule to allow HTTP access to the load balancer on the service port, which defaults to 5555.
+pip install -r requirements.txt
 
-## Data collection details
+export KUBERNETES_API=http://localhost:8001/api/v1
+python collector.py --debug --host=localhost
+```
 
-In "master" mode, the cluster-insight container listens for HTTP requests on local port 5555, as shown in the figure below. In "minion" mode, it listens on local port 4243, and relays requests to the local Docker daemon via the default Unix-domain socket, as described [here](https://docs.docker.com/articles/basics/).
+Running the `pip` and `python` commands above under [virtualenv](https://virtualenv.pypa.io/) is highly recommended, but not required.
 
-![alt text](cluster-insight-architecture.png "cluster-insight service architecture")
+Now the Cluster Insight help page will be available at [http://localhost:5555/](http://localhost:5555/), and if you have `kubectl proxy` running, the REST API will be operational.
 
 ## REST API
 
-These are the APIs that clients can use to get context graphs and raw resource metadata:
+Cluster Insight makes available the following endpoints:
 
+* `/` returns a help page with links to the following.
 * `/cluster` returns a context graph. The format of the context graph is described below.
-* `/cluster/resources/TYPE` returns the raw metadata for all cluster resources of type TYPE, where TYPE is one of { Node, Pod, Service, ReplicationController, Container, Image, Process }.
+* `/cluster/resources` returns all of the resources (nodes), but not the relations (edges).
+* `/cluster/resources/TYPE` returns the raw metadata for all cluster resources of type TYPE, where TYPE is `nodes`, `pods`, `services`, or `rcontrollers`.
```

Comment:
The /cluster/resources/TYPE calls exist (I have been told) in order to enable manual testing of the raw metadata collection upon which the graph construction is based. For example, the collector.get_pods() handler exists only to make it possible to invoke kubernetes.get_pods() independently of the graph construction code. Containers (or images) are different, in that there is no similar kubernetes.get_containers(). Supporting /cluster/resources/containers would require adding code for that purpose, which doesn't seem worthwhile.

To put it another way, the right thing to do would be to remove the support for /cluster/resources/TYPE altogether, but there's no hurry. The same information is available from /get/TYPE on the Kubernetes API.